src/scripts.js

```diff
@@ -10,27 +12,85 @@ import Recipe from './recipe';
 import User from './user';
 import RecipeRepository from './recipeRepository';
 
+const userURL = "http://localhost:3001/api/v1/users";
+const ingredientsURL = "http://localhost:3001/api/v1/ingredients";
+const recipesURL = "http://localhost:3001/api/v1/recipes";
+const retrieveUserData = fetch(userURL)
+.then(response => response.json())
+.then(data => data)
+const retrieveIngredientsData = fetch(ingredientsURL)
+.then(response => response.json())
+.then(data => data)
+const retrieveRecipesData = fetch(recipesURL)
+.then(response => response.json())
+.then(data => data)
+
+let allUsersData = [];
+let allIngredientsData = [];
+let allRecipesData = [];
+
+
+//want to get working off linked app.js
+
 let favButton = document.querySelector('.view-favorites');
 let homeButton = document.querySelector('.home')
 let cardArea = document.querySelector('.all-cards');
-let recipeRepository = new RecipeRepository(recipeData);
-let user, pantry;
 
-window.onload = onStartup();
+
+Promise.all([retrieveUserData, retrieveIngredientsData, retrieveRecipesData])
+  .then((data) => {
+    // console.log(data)
+    allUsersData = data[0]
+    allIngredientsData = data[1]
+    allRecipesData = data[2]
+    onStartup(allUsersData, allIngredientsData, allRecipesData);
```

Comment:
Nice work getting this data stored by putting this function inside of the promsise.